src/neurotechdevkit/scenarios/_time.py

```diff
@@ -4,7 +4,6 @@
 
```

Comment:
```suggestion
from typing import Iterable

```

Reply:
(Wasn't able to add this directly to the line above because of GitHub limitations)